### PR TITLE
Update diagnostic-tools.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@mjrousos 	review and once we flesh out this PR we can merge into your PR and then into master. Excellent work, minor word-smithing.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@mjrousos 	can I merge this into your PR so we can merge to live?

File: aspnetcore/performance/diagnostic-tools.md [hunk 1]
```diff
@@ -11,21 +11,25 @@ uid: performance/diagnostic-tools
...
 
-More information is available in [Visual Studio documentation](https://docs.microsoft.com/visualstudio/profiling/profiling-overview).
+More information is available in [Visual Studio documentation](/visualstudio/profiling/profiling-overview).
 
 ## Application Insights
 
-[Application Insights](https://docs.microsoft.com/azure/application-insights/app-insights-overview) can provide in-depth performance data for your application. Application Insights automatically collects data on response rates, failure rates, dependency response times, and more. You can also log custom events and metrics specific to your app. 
+[Application Insights](/azure/application-insights/app-insights-overview) provides in-depth performance data for your app. Application Insight automatically collects data on response rates, failure rates, dependency response times, and more. Application Insights supports logging custom events and metrics specific to your app.
```
Comment (mjrousos) [inline aspnetcore/performance/diagnostic-tools.md]: Regarding the change to the second sentence here, I think it should remain Application Insights. The name of the service is always plural in my experience.

File: aspnetcore/performance/diagnostic-tools.md [hunk 2]
```diff
@@ -11,21 +11,25 @@ uid: performance/diagnostic-tools
...
 
-When there's a performance issue in production (or even in development or staging environments), Application Insights data is the first place to look to understand what went wrong. Although Application Insights works great in Azure, it can also be used to gather telemetry while running locally from [Visual Studio](https://docs.microsoft.com/azure/application-insights/app-insights-visual-studio) or in other hosting environments. 
+Application Insights is one of the best tools to find performance issues:
```
Comment (mjrousos) [inline aspnetcore/performance/diagnostic-tools.md]: I feel like this doesn't relate to the bullets below it very well. Maybe rephrase this lead in to be something like "Application Insights can be used in many different environments:".

**Comment by Rick-Anderson** [comment]
@mjrousos 	fixed. Let me know when I can merge to your PR and merge to live.
